change-github-default-branch.sh

```diff
@@ -2,17 +2,22 @@
 
 set -euo pipefail
 
-if [ $# -ne 3 ] && [ $# -ne 2 ]; then
-	echo "USAGE: $0 pat_token user/repo [new_default]" 1>&2
+if [ $# -ne 3 ] && [ $# -ne 2 ] && [ -z "$GITHUB_TOKEN" ] && [ $# -ne 1 ]; then
+	echo "USAGE: $0 [pat_token] user/repo [new_default]" 1>&2
```

Comment:
I don't like that the first argument is optional, can we make it second or last?

(oh why oh why didn't I use getopts 😄)

Reply:
I can. There is also a logical error in this if people do supply their token. I'm working on a fix